types/localized-countries/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for localized-countries 1.0
+// Project: https://github.com/marcbachmann/localized-countries#readme
+// Definitions by: coderslagoon <https://github.com/coderslagoon>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface LocalizedCountries {
+    get(code: string): string;
+    array(): Array<{
+        code: string;
+        label: string;
+    }>;
+    object(): { [code: string]: string };
+}
+
+declare function localizedCountries(locale: string|{}): LocalizedCountries;
```

Comment:
Didn't know about the promiscuous nature of `{}` to be honest. Changed it to object and TS 2.2 ... the module simple supports shoving in data from a `require`, with the actual structure of such being private, so `object` seems to be the better choice.